game.js

```diff
@@ -0,0 +1,80 @@
+// get choice of computer in random
+function computerChoice() {
+  const options = ["Rock", "Paper", "Scissors"];
+  const choice = options[Math.floor(Math.random() * options.length)];
+  return choice;
+}
+
+// get the result of round
+function playRound(playerSelection, computerSelection) {
+  if (playerSelection == computerSelection) return "draw";
+  else if (
+    (playerSelection == "Rock" && computerSelection == "Paper") ||
+    (playerSelection == "Paper" && computerSelection == "Scissors") ||
+    (playerSelection == "Scissors" && computerSelection == "Rock")
+  ) return "player_loose";
+  else return "player_win";
+}
+
+// main function
+function game() {
```

Comment:
This function is doing too much things, ideally you should write small function which do only 1 thing and not more.

For example, you can break this function into sub-functions such as:

1) getPlayerName()
2) getPlayerSelection()
3) displayResult(playerName, playerSelection, computerSelection, result)